LibreNMS/Snmptrap/Trap.php

```diff
@@ -144,4 +144,20 @@ public function getRaw()
     {
         return $this->raw;
     }
+
+    /**
+     * Render the Trap for debugging purpose
+     *
+     * @param $detailed
+     * @return string
+     */
+    public function toString($detailed = false)
+    {
+        if ($detailed) {
+            return $this->getTrapOid() . "\n" . json_encode($this->oid_data->reject(function ($value, $key) use ($search) {
```

Comment:
`$search`?

Reply:
copy paste failure ... Let's correct this :)